swat/cas/table.py

```diff
@@ -2784,7 +2783,7 @@ def head(self, n=5, columns=None, bygroup_as_index=True, casout=None):
         columns : list-of-strings, optional
             A subset of columns to return.
         bygroup_as_index : boolean
-            If By groups are specified, should they be converted to an index?
+            When By groups are specified, if bygroup_as_index is True, By groups are converted to an index
```

Comment:
I'm not fond of the wording here.  Maybe "When By groups are specified and bygroup_as_index is True, By groups are converted to an index.  Also, lines should be kept to <78 characters.

Reply:
When by_group_index is True, By groups are converted to an index if they exist